source/common/router/req_header_formatter.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <cstdint>
+#include <iostream>
+#include <string>
+#include <unordered_map>
+
+#include "common/common/logger.h"
+#include "common/http/access_log/access_log_formatter.h"
```

Comment:
we need the #include "envoy/http/access_log.h" header for the RequestInfo class. I can replace the access_log_formatter.h with #include "envoy/http/access_log.h"